cmd/build.go

```diff
@@ -36,15 +38,19 @@ func setupBuildCommand() *cobra.Command {
 func buildCommandAction(cmd *cobra.Command, args []string) error {
 	cmd.Println("Build the package")
 
-	target, err := docs.UpdateReadme()
+	targets, err := docs.UpdateReadmes()
 	if err != nil {
-		return errors.Wrapf(err, "updating %s file failed", docs.ReadmeFile)
+		return errors.Wrap(err, "updating files failed")
 	}
-	if target != "" {
-		cmd.Printf("%s file rendered: %s\n", docs.ReadmeFile, target)
+
+	for _, target := range targets {
+		if target != "" {
```

Comment:
What is the case that the target is empty?

Reply:
It will be empty when README file is static and can't be generated from the template file. This came from `generateReadme(fileName, packageRoot)` function.

Reply:
Ah I see! It might be cleaner to skip this file on the lower layer (in the `docs.UpdateReadmes()`), so "" (empty string) won't leak here.